cmd/key_generate.go

```diff
@@ -276,14 +283,19 @@ Specify a repository to add the GitHub secret and deploy key to:
 
   $ trellis key generate --repo MyOrg/MyBedrockRepo
 
+To skip provisioning, use the --no-provision option:
+  
+  $ trellis key generate --no-provision
+
 Options:
-      --known-hosts Comma-separated list of SSH known hosts (optional)
-      --name        Name of SSH key (Default: trellis_<site_name>_ed25519)
-      --no-github   Skips creating a GitHub secret and deploy key
-      --path        Path for private key (Default: $HOME/.ssh)
-      --provision   Name of environment to provision after key is generated
-      --repo        Repository to add the GitHub secret and deploy key to (Format: OWNER/REPO)
-  -h, --help        show this help
+      --known-hosts  Comma-separated list of SSH known hosts (optional)
+      --name         Name of SSH key (Default: trellis_<site_name>_ed25519)
+      --no-github    Skips creating a GitHub secret and deploy key
+      --no-provision Skips provisioning the environment after key is generated
+      --path         Path for private key (Default: $HOME/.ssh)
+      --provision    Name of environment to provision after key is generated
```

Comment:
```suggestion
      --path         Path for private key (Default: $HOME/.ssh)
      --[no-]provision    Name of environment to provision after key is generated
```
Alternative help output. Thoughts?

Reply:
I'd be okay with that 👍